lib/node_modules/@stdlib/math/base/complex/divide/benchmark/cdiv_smith.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+// MODULES //
+
+var abs = require( '@stdlib/math/base/special/abs' );
+
+// MAIN //
+
+/**
+* Divides two complex numbers.
```

Comment:
You could also just include a benchmark which defines Smith's method and performs the benchmark, rather than have a file. For an example, see `@stdlib/math/base/special/fibonacci` where a variety of implementations are compared, but none are stored as files.